docs/api/qiskit-ibm-runtime/dev/qiskit_ibm_runtime.transpiler.passes.scheduling.PadDynamicalDecoupling.mdx

```diff
@@ -36,52 +36,16 @@ python_api_name: qiskit_ibm_runtime.transpiler.passes.scheduling.PadDynamicalDec
        ("cx", [1, 2], 200), ("cx", [2, 3], 300),
        ("x", None, 50), ("measure", None, 1000)]
   )
-  ```
 
-  ```python
   # balanced X-X sequence on all qubits
   dd_sequence = [XGate(), XGate()]
   pm = PassManager([ALAPScheduleAnalysis(durations),
                     PadDynamicalDecoupling(durations, dd_sequence)])
   circ_dd = pm.run(circ)
-  circ_dd.draw()
+  circ_dd.draw('mpl', style="iqp")
```

Comment:
If we want these to be text based, we need to save the output in a `.. code:: text` block. But I think the image is fine. This text is not very readable to a screen-reader, so the image with alt text will be more useful to non-sighted users.

Reply:
Agreed! I like having an image there rather than the text much more too, but that's nice! TIL